pynvjitlink/VERSION

```diff
@@ -1 +0,0 @@
-0.1.12
```

Comment:
Dropped this file as it appeared to be unused

Perhaps there is some way to have everything [dynamically use this version with `scikit-build-core`]( https://scikit-build-core.readthedocs.io/en/latest/configuration.html#dynamic-metadata ), but wasn't quite seeing how to get that to work without adding more stuff. So just stuck with `pyproject.toml`

Reply:
We should try to adopt the same process as cudf and other libraries that dynamically fetch their version info in pyproject.toml from a VERSION file. It would be ideal for VERSION to be the central source of truth rather than pyproject.toml.

This shows how it works: https://github.com/rapidsai/cudf/blob/aa9d4846e80fad133e7af19aba99cefa04cb8b7b/python/cudf/pyproject.toml#L175

Happy to answer questions on this process if needed.

Reply:
Thanks Bradley! 🙏

The example helped. Have switched things around to keep the `VERSION` file